Makefile

```
@@ -162,14 +162,27 @@ else
 KUSTOMIZE=$(shell which kustomize)
 endif
 
+operator-sdk:
+ifeq (,$(shell which operator-sdk 2>/dev/null))
+	@{ \
+	set -e ;\
+	mkdir -p $(dir $(OPERATOR-SDK)) ;\
+	curl -sSLo $(OPERATOR-SDK) https://github.com/operator-framework/operator-sdk/releases/download/v1.8.0/operator-sdk_$(OS)_$(ARCH) ;\
```

Please add an OPERATOR_SDK_VERSION env var which defaults to the current version we are using (v.1.8.0). This will allow us to experiment with another version when we need to 

```suggestion
	curl -sSLo $(OPERATOR-SDK) https://github.com/operator-framework/operator-sdk/releases/download/$(OPERATOR_SDK_VERSION)/operator-sdk_$(OS)_$(ARCH) ;\
```